sdk/src/Services/S3/Custom/Transfer/_bcl+netstandard/ITransferUtility.async.cs

```diff
@@ -145,6 +145,94 @@ public partial interface ITransferUtility
         /// <returns>The task object representing the asynchronous operation.</returns>
         Task UploadDirectoryAsync(TransferUtilityUploadDirectoryRequest request, CancellationToken cancellationToken = default(CancellationToken));
 
+        /// <summary>
```

Comment:
i updated the files that the existing uploaddirectory operations were in. i think theres also transfer utility files in the non `bcl+standard` folder too, but since these are partial classes, they will all be compiled together. so the main thing to update is both the `sync` + `async`